ubiquity/dell-recovery.py

```diff
@@ -125,6 +125,8 @@ def prepare(self, unfiltered=False):
         """Prepares the debconf plugin"""
         destination = self.db.get('dell-recovery/destination')
         self.ui.set_type(destination)
+        if 'UBIQUITY_AUTOMATIC' in os.environ:
```

Comment:
I think you should leave a comment above this explaining this is meant for development purposes

Reply:
Re-check the code, and found another place to modify to make it more nature.